docs/helpers.md

```diff
@@ -0,0 +1,75 @@
+# Helper models, views, and ModelAdmins
+
+## OnlineBase
+
+`cms.models.OnlineBase` provides publication controls.
+It provides a single field: `is_online`. When it is `False` (unchecked), it will not appear on the front-end of your website.
+
+You do not have to do anything other than inherit from OnlineBase for this to work; objects that are not online are automatically excluded from querysets via a custom manager, `OnlineBaseManager`.
+We will go into more detail about that [elsewhere](publication-control.md).
+
+`OnlineBase` implements a `get_preview_url()` method that generates a secret URL at which an object can be previewed by non-administrative users.
+We use this in our [OSM Jet](https://github.com/onespacemedia/osm-jet) admin skin to generate a "click to copy public preview URL" function.
+If your model does not implement a `get_absolute_url()` method, this method will return `None`.
+
+`cms.admin.OnlineBaseAdmin` is the companion `ModelAdmin` for this model; derivatives of OnlineBase should probably inherit from this for their user model. It defines `PUBLICATION_FIELDS`, which you can use on your model's admin thusly:
+
+```python
+class YourModelAdmin(OnlineBaseAdmin):
+    fieldsets = [
+        # ... your fieldsets here ...
+        OnlineBaseAdmin.PUBLICATION_FIELDS,
+    ]
+```
+
+`OnlineBaseAdmin` also adds a helpful list `action`: you will be able to turn off items as a batch from the list view.
+
+By default, new objects will have the checkbox for `is_online` checked by default in the admin, if your `ModelAdmin` extends from `OnlineBase`.
+If you want to change this, set `ONLINE_DEFAULT` to `False` in your Django settings.
+
+## PageBase
+
+`cms.models.PageBase` provides the publication control that `OnlineBase` does, but also contains many useful fields that are fairly typical for a typical article-like thing on a website.
+If an instance of your model lives at its own URL and has a title, such as a news article or a blog post, then you almost certainly want to inherit from `PageBase`.
+The CMS's own `Page` model inherits from it.
+
+PageBase defines several fields that make this useful for article-like things:
+
+* A title and slug
+* SEO controls: title tag override, meta description, robots controls
+* Fields for Twitter and Facebook (OpenGraph) cards
+
+`cms.admin.PageBaseAdmin` is the companion `ModelAdmin`; at risk of repeating ourselves, anything that derives from `PageBase` definitely wants to use `PageBaseAdmin`.
```

Comment:
"at risk of repeating ourselves" I don't think you need to excuse this at all - it's good to repeat things if it make it clearer